src/components/tile-implementations/earthquake/useEarthquakeApi.test.tsx

```
@@ -0,0 +1,152 @@
+    {children}
+  </MockDataServicesProvider>
+);
+
+describe('useEarthquakeApi', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+    storageManager.clearTileState(); // Clear cache between tests
+  });
+
+  it('fetches and maps USGS earthquake data successfully', async () => {
+    (globalThis.fetch as unknown as { mockResolvedValueOnce: Function }).mockResolvedValueOnce({
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)